src/Main.tsx

```diff
@@ -1,15 +1,26 @@
-import { getChainOptions, WalletProvider } from '@terra-money/wallet-provider';
-import { ConnectSample } from 'components/ConnectSample';
-import { QuerySample } from 'components/QuerySample';
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './main.css';
+import { getChainOptions, WalletProvider } from "@terra-money/wallet-provider";
+import { RecoilRoot } from "recoil";
+import { ConnectSample } from "components/ConnectSample";
+import { QuerySample } from "components/QuerySample";
+import { useInitAddress, useInitNetwork } from "data/init";
+import React from "react";
+import ReactDOM from "react-dom";
+import "./main.css";
+
+const RecoilInit = () => {
+  useInitAddress();
+  useInitNetwork();
+  return <></>;
+};
 
 export function Main() {
   return (
-    <div className="main">
-      <ConnectSample />
-      <QuerySample />
-    </div>
+    <RecoilRoot>
```

Comment:
Should this wrap the entire app similar to `WalletProvider` in index.tsx?

Or is this to create a separate state for the Liquidation Withdrawal fiunctionality?